server.js

```diff
@@ -3,28 +3,29 @@
  * the verification process may break
  *******************************************/
 
-var express = require('express');
-var app = express();
-try{
-  var mongoose = require('mongoose');
```

Comment:
I noticed that changing this from var to const broke the test for me when running locally. Were you able to get this to pass? Basically, I forked this repo and checked out this branch, got it running - tried to do the first lesson and it didn't pass. The test goes to `URL/_api/is-mongoose-ok` and it was giving me back a `mongoose is not defined`, when I changed this back to `var` it gave the expect `isMongooseOk: true`. 

I'm curious if you want to give that a try @Sky020 and see if you have the same issue. If you do, I'm thinking we should maybe not change these files on the boilerplates and leave them how they were. That would probably go for the rest of them as well.

Reply:
@moT01 Thank you, for catching that. It was poor diligence on my part, and appear to have missed a few things.

I will push the necessary changes, soon.